event-streams/cr-examples/kafka-mirror-maker-2/kafka-mirror-maker-2-non-production.yaml

```diff
@@ -17,21 +17,10 @@ spec:
           productName: IBM Event Streams for Non Production
           productVersion: EVENTSTREAMS_PRODUCT_VERSION
           productMetric: VIRTUAL_PROCESSOR_CORE
-          productChargedContainers: <ADD-NAME-OF-MM2-CR>-mirrormaker2
+          productChargedContainers: "All"
           cloudpakId: c8b82d189e7545f0892db9ef2731b90d
           cloudpakName: IBM Cloud Pak for Integration
           productCloudpakRatio: "2:1"
-      affinity:
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-              - matchExpressions:
-                  - key: kubernetes.io/arch
-                    operator: In
-                    values:
-                      - amd64
-                      - s390x
-                      - ppc64le
```

Comment:
Why are we removing the `affinity` section? It's present in the OpenShift ALM sample.